src/wymeditor/iframe/default/wymiframe.html

```diff
@@ -21,5 +21,5 @@
 <title>WYMeditor iframe</title>
 <link rel="stylesheet" type="text/css" media="screen" href="wymiframe.css" />
 </head>
-<body class="wym_iframe"></body>
+<body></body>
```

Comment:
With [this change](https://github.com/wymeditor/wymeditor/pull/624/files#r21754781), you modified the way `wym_iframe` is required and used. I'm trying to understand why you did that and how it will affect existing custom iframes.